web/client/components/map/openlayers/plugins/VectorLayer.js

```diff
@@ -106,23 +106,7 @@ var styleFunction = function(feature) {
 
 Layers.registerType('vector', {
     create: (options) => {
-        let features;
```

Comment:
This is needed to allow configuring the layer via JSON instead of using Feature.jsx components